docs/source/index.rst

```diff
@@ -0,0 +1,20 @@
+
+Welcome to PodmanPy's documentation!
+====================================
+
+.. automodule:: podman
+    :members:
+    :inherited-members:
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+
+
+Indices and tables
+==================
+
+* :ref:`genindex`
+* :ref:`modindex`
+* :ref:`search`
```

Comment:
I've got a warning that modules.rst isn't referenced in toctree. Not certain if it's an issue or if we should add it here.

/home/aschultz/Documents/workspace/containers/podman-py/build/docs/source/modules.rst: WARNING: document isn't included in any toctree

Reply:
added